docs/creating-extensions.md

```diff
@@ -134,6 +135,7 @@ Use `%env(VAR_NAME)%` syntax in service configuration to reference environment v
 ## Configuration Reference
 
 **Package Type:** `ai-mate-extension` (required)
+FIXME: Should we really require this? What if a package (library) want to have one extra class for the mcp?
```

Comment:
i would say for now yes! mixing that would bring more questions in. e.g. can i reuse my bundle services.

It also makes our code way easier for now